src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2348,22 +2348,8 @@ private static byte[] ConvertEnvVarsToByteArray(StringDictionary sd)
             return bytes;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Start-Process.UseNewEnvironment parameter should reset environment variables for child process
```powershell
Expected strings to be the same, but they were different.
Expected length: 2
Actual length:   1
Strings differ at index 1.
Expected: '1;'
But was:  '1'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1: line 188
188:             Get-Content -LiteralPath $outputFile | Should -BeExactly "1;$userName"
```